yt_dlp/extractor/goplay.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+from .common import InfoExtractor
+from ..utils import int_or_none
+from .goplay_auth_aws import AwsIdp
+
+
+class GoPlayIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?goplay\.be/video/(?P<series>[^/]+)/(?P<season>[^/]+)(/(?P<episode>[^/]+)|)'
```

Comment:
Add a test for the case with only 2 parts

Reply:
That case actually does not exist. After quite some searching I found an item that has a single part though: https://www.goplay.be/a-family-for-the-holidays.

I've updated the regexp to not make the last part optional but instead make the series and season parts optional. If they're not part of the URL the code now checks if it's a movie, and if so downloads it correctly.